addons/pos_hr/static/src/overrides/components/cashier_name/cashier_name.js

```diff
@@ -25,4 +25,7 @@ patch(CashierName.prototype, {
         }
         return super.cssClass;
     },
+    get username() {
+        return this.pos.cashier?.employee?.name || "";
```

Comment:
Not necessarily true when the module is installed but the setting is not checked